agent/consul/leader_connect_test.go

```diff
@@ -36,7 +36,7 @@ func TestConnectCA_ConfigurationSet_ChangeKeyConfig_Primary(t *testing.T) {
 		keyBits int
 	}{
 		{connect.DefaultPrivateKeyType, connect.DefaultPrivateKeyBits},
-		{"ec", 256},
+		// {"ec", 256}, skip since values are same as Defaults
```

Comment:
Should we just delete this line then?

Reply:
I wanted to leave it in case the Defaults ever changed in the future (in which case we might want to un-comment)

Reply:
👍